.github/actions/maven-build-and-tag/action.yml

```diff
@@ -9,6 +9,10 @@ inputs:
     description: The Java distribution to perform the build
     required: false
     default: 'temurin'
+  build-cache-key:
```

Comment:
I wonder if a better name should be used to avoiding this cache key with the standard maven cache (`diff-build-cache-key` (?))

Reply:
In any case it would be nice to document on the README that this input should be used in combination with the maven-build-cache-extension features